cellar_rebalancer/src/config.rs

```diff
@@ -65,6 +67,24 @@ impl Default for CellarRebalancerConfig {
         }
     }
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct TlsSection {
+    pub server_cert: String,
```

Comment:
Nit: prefer to suffix strings meant to be paths with `_path` to indicate it's not a serialization or something.